src/ipfs.io/docs/install/index.html

```diff
@@ -59,9 +59,9 @@
 
 Dependencies:
 
-- [Install Go 1.4](https://golang.org/doc/install) (to compile. [make sure it's 1.4](docs/install#check-go-version))
-- [Install Git](https://git-scm.com/) (to download dependencies)
-- *Optionally*, install [FUSE](docs/install#install-fuse) (for mounting / advanced use)
+- [Install Go 1.4](https://golang.org/doc/install) (to compile. [make sure it's 1.4](#check-go-version))
+- [Install Git](http://git-scm.com/) (to download dependencies)
```

Comment:
what's wrong with `https`? we have `https` now, too. should prob keep it as https.